app/assets/translations/appConstants/bebbo/RSen.ts

```diff
@@ -1,453 +1,458 @@
+                "walkthroughTextstitle1": "Games ",
+                "walkthroughTextssubtitle1": "to help your child learn new things everyday",
+                "walkthroughTextstitle2": "Tools",
+                "walkthroughTextssubtitle2": "to track your child's development, growth, vaccines and health",
+                "walkthroughTextstitle3": "Advice",
+                "walkthroughTextssubtitle3": "tailored to your questions and the age of your child",
+                "walkthroughButtonBack": "Back",
+                "walkthroughButtonNext": "Next",
+                "walkthroughButtonSkip": "Skip",
+                "tNcheader": "Terms and Conditions",
+                "tncScreenContent": "Bebbo does not collect or store any personal information. All the information you enter is saved only on your phone.",
+                "tNccheckbox1": "I have read, understood and accepted the Terms and Conditions of the Bebbo application",
+                "tncCheckBoxText": "Terms and Conditions of the Bebbo application",
+                "tNccheckbox2": "I have read, understood and accepted the ",
+                "tNcprivacyPolicy": "Privacy Policy of the Bebbo application",
+                "tNccheckbox3": "I understand that my usage of the app may generate anonymized data, which may be used to improve Bebbo. While providing this data is not mandatory, I recognize that it can help enhance the overall user experience.",
+                "tNcprivacyPolicyTitle": "Privacy Policy",
+                "tNcacceptbtn": "I accept the Terms and Conditions",
+                "loadingText": "Fetching content…",
+                "childSetupheader": "Please take a moment to personalize your app",
+                "childSetupdobLabel": "Child's date of birth",
+                "childSetupdobSelector": "Enter date of birth / expected due date",
+                "childSetupdueLabel": "Original due date",
+                "childSetupdueSelector": "Enter original due date",
+                "childSetupprematureLabel": "Baby born prematurely",
+                "childSetupprematureMessage": "A baby born before 37 weeks of pregnancy is considered premature or born too early.",
+                "childSetuprelationSelectTitle": "Your relationship to the child",
+                "childSetuprelationSelectText": "Select",
+                "childSetupcontinueBtnText": "Continue",
+                "childSetupListheader": "Your child's profile has been created!",
+                "childSetupListsubHeader": "Please review and add siblings if you have other children between 0-6 years old. ",
+                "childSetupListaddSiblingBtn": "Add sister or brother",
+                "childSetupListeditSiblingBtn": "Edit child's profile",
+                "childSetupListcontinueBtnText": "Continue",
+                "childSetupListsaveBtnText": "Save data",
+                "drawerMenuchildInfo": "Born on {{childdob}} ",
```

Comment:
Was extra space at the end intentional?

Reply:
We can update this in string changes. kindly create one task for all string changes.